packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -195,9 +202,12 @@ export class SceneTreeTableCell {
   }
 
   private displayValue = (): string => {
-    return this.value == null || this.value.replace(' ', '') === ''
-      ? this.placeholder
-      : this.value;
+    const resp =
+      this.value == null || this.value.replace(' ', '') === ''
```

Comment:
Not your change, but while we're here... `this.value.replace(' ', '')` should prob be `.trim()`.